### DISCUSSION TITLE
Mapping right Cmd to Ctrl key (macOS). Possible?

### DISCUSSION THREAD

**Discussion by VKondakoff** [kovidgoyal/kitty#4139] [Q&A]
Hi!

I was hoping to be able to remap right Cmd key to Ctrl key (macOS). 
`kitty --debug-input` shows right Cmd as `0x36` and Ctrl as `0x3b`, but this simple line in Kitty config does not seems to work: `map 0x36 0x3b`.
Is it possible? Can you, please, share an example?

Thanks!

**Comment by Luflosi** [comment]
See https://github.com/kovidgoyal/kitty/issues/1077.

**Comment by page-down** [comment]
You may want to try Karabiner-Elements, which allows you to customize the key definitions for each keyboard device in different applications.
https://github.com/pqrs-org/Karabiner-Elements

It is also possible to search and import complex definition rules contributed by the community, with a single click.
https://ke-complex-modifications.pqrs.org/
